mne/evoked.py

```diff
@@ -748,6 +804,8 @@ def __init__(self, data, info, tmin=0., comment='', nave=1, kind='average',
             raise ValueError('unknown kind "%s", should be "average" or '
                              '"standard_error"' % (self.kind,))
         self._aspect_kind = _aspect_dict[self.kind]
+        self.baseline = baseline
+        self.apply_baseline(self.baseline, verbose=verbose)
```

Comment:
apply_baseline should update self already

Reply:
Yes, but I think this additional line 807 makes it more explicit that we're storing the `baseline` parameter in `self.baseline`. If you insist, I can remove it; personally I find it nicer the way it is now.

Reply:
I've tried to improve things in def384d